### ISSUE TITLE
[Bug] Axios invalid URL

### ISSUE THREAD

**Issue by code-farmer-i** [apache/skywalking#10070]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

NodeJS Server Side Agent (apache/skywalking-nodejs)

### What happened

Throw invalid URL when I use axios

### What you expected to happen

no exception

### How to reproduce

```js
const demoServer = axios.create({
  baseURL: 'http://test.com.cn',
});

demoServer.request({
    method: 'post',
    url: '/api/demoApi',
  });
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Our axios test is like this, https://github.com/apache/skywalking-nodejs/blob/master/tests/plugins/axios/server.ts

**Comment by code-farmer-i** [comment]
> Already fixed by [apache/skywalking-nodejs#93](https://github.com/apache/skywalking-nodejs/pull/93). Let me initiate 0.6.0 release

When is the 0.6.0 release expected
